src/lib/misc.c

```diff
@@ -602,12 +607,15 @@ pgp_memory_pad(pgp_memory_t *mem, size_t length)
 \param src Data to add
 \param length Length of data to add
 */
-void
+int
```

Comment:
@ronaldtse yep, at [this commit](https://github.com/riboseinc/rnp/pull/223/commits/96a2f5b58ce8528d6943c7fed9488bb44d8a044a)